modules/support-reviewing-an-access-request-from-the-hybrid-console.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+// * osd_architecture/osd-support.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="support-reviewing-an-access-request-from-the-hybrid-cloud-console_{context}"]
+= Reviewing an access request from the {hybrid-console-second}
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `Cluster Owner` role.
+endif::openshift-rosa,openshift-dedicated[]
+
+
+.Procedure
+
+. Navigate to {cluster-manager-url} and select *Clusters*.
+
+. Click the cluster *Name* to review the *Access Request*.
```

Comment:
Just checking if "Name" really needs to be bolded here, or if it's just choosing a cluster from the list, and could maybe just be normal "name" text?